src/ucp/rndv/proto_rndv.c

```diff
@@ -697,6 +715,10 @@ ucp_proto_rndv_handle_rtr(void *arg, void *data, size_t length, unsigned flags)
             goto err_request_fail;
         }
     } else {
+        ucs_assertv(&req->send.state.dt_iter.dt_class == UCP_DATATYPE_CONTIG,
+                    "fragmented rndv is not supported with datatype %s",
```

Comment:
`rendezvous` is more common in messages for users
```suggestion
                    "fragmented rendezvous is not supported with datatype %s",
```